roles/satellite-clone/tasks/pre_install_check.yml

```diff
@@ -17,3 +14,35 @@
 - name: Check if root has enough space
   fail: msg="Your root partition should atleast have {{ required_root_free_space }}GB but it contains only {{ root_free_space }}GB"
   when: root_free_space | int <= required_root_free_space | int
+
+- name: Check for config tar file
+  stat:
+    path: '{{ backup_dir }}/config_files.tar.gz'
+    get_checksum: False
+    get_md5: False
+  register: config_data
+
+- name: Fail if the config tar file is not present or not accessible
+  fail: msg="{{ backup_dir }}/config_files.tar.gz is not present or not accessible"
+  when: not config_data.stat.exists
+
+- name: Check the presence of pulp tar file
+  stat:
+    path: '{{ backup_dir }}/pulp_data.tar'
+    get_checksum: False
+    get_md5: False
+  register: pulp_data
+
+# These variables will be set to True as required in backup_check.yml as necessary
+- name: Set all backup types to False
+  set_fact:
+    online_backup: False
+    standard_backup: False
+
```

Comment:
I moved these methods to a separate check because this file can be skipped. The current task requires these variables to be registered (like pulp_data) or the tasks will fail, so I didn't want to give the user an option to skip this.